mwdb/resources/__init__.py

```diff
@@ -20,7 +20,7 @@ def decorator(f):
         def endpoint(*args, **kwargs):
             for required_cap in required_caps:
                 if not g.auth_user.has_rights(required_cap):
-                    raise Forbidden("You are not permitted to perform this action")
+                    raise Forbidden("You are not permitted to perform action: " + required_cap)
```

Comment:
Capabilities not always map well to the actual action. Maybe something like that would be better:
```suggestion
                    raise Forbidden(f"You don't have required capability ({required_cap}) to perform this action")
```